tests/test_transformDiaSourceCatalog.py

```diff
@@ -182,21 +182,25 @@ def test_bit_unpacker(self):
             else:
                 self.assertFalse(flag['base_PixelFlags_flag_offimage'])
 
+    def test_flag_existence_check(self):
+        unpacker = UnpackApdbFlags(self.config.flagMap, "DiaSource")
+
+        self.assertTrue(unpacker.flagExists('base_PixelFlags_flag'))
+        self.assertFalse(unpacker.flagExists(''))
+        with self.assertRaises(ValueError):
+            unpacker.flagExists(['base_PixelFlags_flag'], columnName='doesNotExist')
```

Comment:
The first argument here is a `list`, but should just be a string.

I'd also suggest `assertRaisesRegex` with some part of the expected error message, to ensure it really is the exception you meant check for.